rs-week1/src/App.tsx

```diff
@@ -1,34 +1,29 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import './App.css'
+import React, { useCallback, useState } from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Home from './components/pages/Home';
+import Header from './components/pages/Header';
+import About from './components/pages/About';
+import Page404 from './components/pages/Page404';
+import FormPage from './components/pages/formPage/FormPage';
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [namePage, setNamePage] = useState<string>('');
+
+  const handleNamePage = useCallback((namePage: string) => {
+    setNamePage(namePage);
+  }, []);
 
   return (
-    <div className="App">
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src="/vite.svg" className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://reactjs.org" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </div>
-  )
+    <>
+      <Header namePageTerm={namePage} />
+      <Routes>
+        <Route path="/" element={<Home onChangeNamePage={handleNamePage} />} />
+        <Route path="/about" element={<About onChangeNamePage={handleNamePage} />} />
```

Comment:
```suggestion
        <Route path="/about" element={<About />} />
```
Здесь также как и выше

Reply:
Сделано